frontend/packages/client/src/pages/Community.js

```diff
@@ -199,13 +205,14 @@ export default function Community() {
   const notMobile = useMediaQuery();
 
   const onUserLeaveCommunity = async () => {
-    // if current user leaving community is admin or author
-    // trigger update on admin and author list
-    if (authors?.find((author) => author.addr === addr)) {
-      await reFectAuthors();
+    if (members?.find((member) => member.addr === addr)) {
+      await queryClient.invalidateQueries(queryKey);
     }
-    if (admins?.find((admin) => admin.addr === addr)) {
-      await reFectAdmins();
+  };
+
+  const onUserJoinCommunity = async () => {
+    if (members?.length < 6) {
+      await queryClient.invalidateQueries(queryKey);
     }
```

Comment:
if there are less than 6 avatars on community page there's room to show another one, pull members again and show then